source/fundamentals/logging.txt

```diff
@@ -0,0 +1,370 @@
+.. _golang-logging:
+
+=======
+Logging
+=======
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Starting in version 1.12, the {+driver-short+} provides a logging
+interface to record driver events. In this guide, you can learn how to
+use the driver to configure logging for your application.
+
+Enable Logging
+--------------
+
+To configure a logger on your ``Client`` instance, call the
+``SetLoggerOptions()`` method when specifying your ``ClientOptions``
+object and pass these options to the ``Connect()`` method. The
```

Comment:
good call. Do you think the examples further down on this page need the `Connect()` call? I'm wondering if it's unneeded since the page is really about the `LoggerOptions` type.